pkg/node/volumemgr_test.go

```diff
@@ -2125,3 +2125,45 @@ func TestVolumeManager_failedToRemoveFinalizer(t *testing.T) {
 	_, err = vm.removeFinalizer(context.TODO(), volumeCR)
 	assert.NotNil(t, err)
 }
+
+func TestVolumeManager_retryDriveUpdate(t *testing.T) {
```

Comment:
Can we also check via expectEvent if the event was raised?

Reply:
done